pdr_backend/pdr_dashboard/pages/home.py

```diff
@@ -0,0 +1,158 @@
+from dash import html, dcc
+import dash_bootstrap_components as dbc
+from pdr_backend.pdr_dashboard.dash_components.view_elements import (
+    get_table,
+    get_feeds_data,
+    get_predictoors_data,
+    get_metric,
+    get_date_period_selection_component,
+)
+from pdr_backend.pdr_dashboard.dash_components.util import (
+    get_feed_ids_based_on_predictoors_from_db,
+)
+
+
+def layout(app):
+    return html.Div(
+        [
+            dcc.Store(id="user-payout-stats"),
+            dcc.Loading(
+                id="loading",
+                type="default",
+                children=get_main_container(app),
+                custom_spinner=html.H2(dbc.Spinner(), style={"height": "100%"}),
+            ),
+        ],
+        style={"height": "100%"},
+    )
+
+
+def get_main_container(app):
+    return html.Div(
+        [get_input_column(app), get_graphs_column()],
+        className="main-container",
+    )
+
+
+def get_graphs_column():
+    return html.Div(
+        [get_graphs_column_metrics_row(), get_graphs_column_plots_row()],
+        id="graphs_container",
+        style={
+            "height": "100%",
+            "width": "70%",
+            "display": "flex",
+            "flexDirection": "column",
+            "justifyContent": "start",
+            "paddingLeft": "50px",
+        },
+    )
+
+
+def get_input_column(app):
+    feed_cols, feed_data = get_feeds_data(app)
+    predictoor_cols, predictoor_data = get_predictoors_data(app)
+
+    selected_predictoors = list(range(len(app.favourite_addresses)))
+
+    if app.favourite_addresses:
+        feed_ids = get_feed_ids_based_on_predictoors_from_db(
+            app.lake_dir,
+            app.favourite_addresses,
+        )
+
+        if feed_ids:
+            feed_data = [
+                feed for feed in app.feeds_data if feed["contract"] in feed_ids
+            ]
+
+        selected_feeds = list(range(len(feed_ids)))
+    else:
+        selected_feeds = []
+
+    return html.Div(
+        [
+            html.Div(
+                get_table(
```

Comment:
I also think by this point we can have get_predictoors_table and get_feeds_table. The "get_table" function used to be simpler buy now it's reached 7 parameters and is almost a pass-through to the table itself

Reply:
In my opinion, the `get_table` method has too many components and it's worth generalizing them rather than repeating them, it also contains other components beside the table, like inputs